### PR TITLE
Fix markdown duplicates

### PR DESCRIPTION
**Pull request by lukmdo**
moves all md files to `markdown` dir
- sync maintainers (was not in sync)
- `ln -s` for README 

relates https://github.com/OpenObservability/OpenMetrics/pull/102

### CODE REVIEW

**Comment by lukmdo** [comment]
See https://github.com/lukmdo/OpenMetrics/tree/fix_md_duplicates

File: markdown/README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ data, influenced by the [Prometheus exposition
 format](https://prometheus.io/docs/instrumenting/exposition_formats/).
 
 Discussion of the standard is happening in the
-[issues](https://github.com/RichiH/OpenMetrics/issues) on this repository and in bi-weekly meetings; invitation upon request and after basic vetting.
+[issues](https://github.com/OpenObservability/OpenMetrics/issues) on this repository and in bi-weekly meetings; invitation upon request and after basic vetting.
```
Comment (lukmdo) [inline markdown/README.md]: this is best to stay absolute I believe  

better then "/../../issues" IMO

**Comment by lucperkins** [comment]
@lukmdo It looks like I added `CONTRIBUTORS.md`, `README.md`, and `MAINTAINERS.md` to the `markdown` folder by mistake (through an errant `mv *.md markdown` apparently). So actually those should be removed from the `markdown` folder rather than from the main directory.

The purpose of the `markdown` folder is to hold Markdown files that will eventually be added to the website (currently under development). So the only files that should be in that folder are `metric_exposition_format.md` and `protobuf_vs_text.md`. Could you update the PR with those changes? I can also submit a follow-up PR if you prefer.

**Comment by lukmdo** [comment]
@lucperkins I see - done. Then if OK
https://github.com/OpenObservability/OpenMetrics/pull/102 could be closed

**Comment by lucperkins** [comment]
👍  from me

**Pull request merged**